crates/rolldown/src/bundler.rs

```diff
@@ -55,24 +55,24 @@ impl Bundler {
 impl Bundler {
   #[tracing::instrument(level = "debug", skip_all, parent = &self.session_span)]
   pub async fn write(&mut self) -> BuildResult<BundleOutput> {
-    trace_action!(action::BuildStart { kind: "BuildStart" });
+    trace_action!(action::BuildStart { action: "BuildStart" });
```

Comment:
This seems a little redundant in rust? Since you already represent the `action` with `taggedUnion`